frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -122,7 +127,7 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
 
   return (
     <PointClusterer
-      points={features}
+      points={[...features, ...draftProperties]}
```

Comment:
does this allow there to be a bunch of draft pins grouped together if someone is creating multiple buildings on a parcel?

Reply:
this will append all of the draft properties to the list of regular properties.